Packs/Box/ReleaseNotes/3_1_23.md

```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### BoxEventCollector Parsing Rule
+
+- Updated the Parsing Rule logic.
```

Comment:
```suggestion
Updated the Parsing Rule logic.
```
When the Release notes have a single line, you do not use a "-".  https://xsoar.pan.dev/docs/documentation/release-notes